Cargo.toml

```
@@ -105,7 +105,7 @@ criterion = "0.3"
 delay_map = "0.3"
 derivative = "2"
 dirs = "3"
-discv5 = { git="https://github.com/sigp/discv5", rev="87a0260db8a27595d01999ea5850848c43cc1bf0", features = ["libp2p"] }
+discv5 = { git="https://github.com/sigp/discv5", rev="1b5218d01015023f32e12c0a9ac6780759a550bf", features = ["libp2p"] }
```

```suggestion
discv5 = { version = "0.4.0", features = ["libp2p"] }
```
age released a new version Pawan

---

Discv5 has the `libp2p` feature for using `PeerId`s, so we need to update the libp2p version in discv5's Cargo.toml as well
We were using the [`lighthouse`](https://github.com/sigp/discv5/commit/87a0260db8a27595d01999ea5850848c43cc1bf0)
I upgraded it to use the branch I made (https://github.com/sigp/discv5/tree/lighthouse-with-yamux), this branch builds upon version 0.4.0